lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -154,6 +155,8 @@ def check_alt_backend(alt_backend):
 @pytest.mark.parametrize("toolbar", ["toolbar2", "toolmanager"])
 @pytest.mark.flaky(reruns=3)
 def test_interactive_backend(backend, toolbar):
+    if platform.machine() == 'aarch64':
+        pytest.skip("Skipping for aarch64 architecture")
```

Comment:
This still doesn't explain _why_.

Reply:
As per @tacaswell's suggestion, we are skipping the installation of 'pandas' & 'jupyter' and skipping related tests for aarch64 platform. Skipping this because it was getting failed without those dependencies.

Reply:
Then it should only be skipped on [CI](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) _and_ this platform, not just this platform. It works fine on aarch64 otherwise.

Reply:
Correct, Thanks for pointing it out, Modified the same. Now it will skip for aarch64 platform on Travis only. Please check.